corehq/apps/users/migrations/0043_add_release_management_permission.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+
+from corehq.apps.users.models_role import SQLPermission
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def create_release_management_permission(apps, schema_editor):
+    SQLPermission.objects.get_or_create(value='access_release_management')
+
```

Comment:
This felt a bit strange, but we don't plan to add this permission to any existing roles so I think this is correct.

Reply:
You could even do `SQLPermission.create_all()`